kafka/kafka-2.1.1/image.yaml

```diff
@@ -15,22 +15,12 @@ labels:
   - name: "io.openshift.tags"
     value: "messaging,amq,jboss"
 
-envs:
-  - name: "KAFKA_HOME"
-    value: "/opt/kafka"
-  - name: "STUNNEL_HOME"
-    value: "/opt/stunnel"
-  - name: "S2I_HOME"
-    value: "/opt/kafka/s2i"
-  - name: "COM_REDHAT_COMPONENT"
-    value: "amqstreams12-kafka-21-openshift-container"
-
 modules:
   repositories:
     - name: cct_module
       git:
         url: https://github.com/jboss-openshift/cct_module.git
-        ref: sprint-28
+        ref: sprint-35
```

Comment:
This is how cct_modules has been tagging releases. Fortunately, I hope to strip these out entirely anyway in a future PR